docs/howto/overriding-templates.txt

```diff
@@ -0,0 +1,88 @@
+====================
+Overriding Templates
+====================
+
+Overriding from the project's templates directory
+=================================================
+
+To override templates from a package in ``django.contrib`` or from a third-party
+package, the preferred method is to create replacement templates in your
```

Comment:
preferred to who? Maybe something "A common way to override templates.... is to create replacement templates..." would be better.